### PR TITLE
Here is some Nick Cage - Closes #2

### LINKED ISSUE #2: Needs More Nick Cage 🕺🏻
Our client likes the site, but wants us to have `More Nick Cage` before our first release. We need:

 - [x] Create a new `html` file called 'nick.html'.
 - [x] Grab two photos from [PlaceCage](https://www.placecage.com/) and put them into the file
 - [x] Merge the file into master

Please submit your PR with a checklist, showing that you have done these three things. BONUS: Squash your commits.

### PR DESCRIPTION
**Pull request by shanselman**
- [x] Create a new `html` file called 'nick.html'.
 - [x] Grab two photos from [PlaceCage](https://www.placecage.com/) and put them into the file
 - [x] Merge the file into master

This PR should close #2

### CODE REVIEW

**Comment by VerteDinde** [comment]
Hey @caterinasworld - would you mind reviewing this PR?

File: nick.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<h1>More Nick Cage</h1>
+
+<img src="https://www.placecage.com/c/400/300">
```
Comment (shanselman) [inline nick.html:3]: Loving this one.

Comment (caterinasworld) [reply nick.html:3]: 💯

File: nick.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+
+<img src="https://www.placecage.com/g/400/500">
```
Comment (shanselman) [inline nick.html:6]: LGTM :shipit:

**Review by caterinasworld — APPROVED** [review]

**Pull request merged**